core/src/integration-test/java/com/scalar/db/storage/objectstorage/ConsensusCommitAdminIntegrationTestWithObjectStorage.java

```diff
@@ -0,0 +1,138 @@
+package com.scalar.db.storage.objectstorage;
+
+import com.scalar.db.transaction.consensuscommit.ConsensusCommitAdminIntegrationTestBase;
+import com.scalar.db.util.AdminTestUtils;
+import java.util.Properties;
+import org.junit.jupiter.api.Disabled;
+
+public class ConsensusCommitAdminIntegrationTestWithObjectStorage
+    extends ConsensusCommitAdminIntegrationTestBase {
+
+  @Override
+  protected Properties getProps(String testName) {
+    return ObjectStorageEnv.getProperties(testName);
+  }
+
+  @Override
+  protected AdminTestUtils getAdminTestUtils(String testName) {
+    return new ObjectStorageAdminTestUtils(getProperties(testName));
+  }
+
+  @Override
+  @Disabled("Temporary disable because it includes DML operations")
```

Comment:
Depending on which stage this message is meant for, should it be the following instead?:

```suggestion
  @Disabled("Temporarily disabled because it includes DML operations")
```

Also, maybe we should specify what is temporarily disabled here.

> [!NOTE]
>
> Any change we make here should be applied in other files where this wording is used.